src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/security/authorization/Profile.kt

```diff
@@ -13,7 +13,11 @@ val CommonProfile.missingPermissions: MutableSet<ReifiedPermission>
 
 var CommonProfile.orderlyWebPermissions: PermissionSet
     get() = this.getAttributeOrDefault(PERMISSIONS, PermissionSet())
-    set(value) = this.addAttribute(PERMISSIONS, value)
+    set(value)
+    {
+        this.removeAttribute(PERMISSIONS)
+        this.addAttribute(PERMISSIONS, value)
```

Comment:
Do you know what, I thought this would throw an error if the key exists already but looks like it's a Java HashMap `put` operation which actually does replacement. So this can be removed.